portfolio/src/main/webapp/blog.html

```diff
@@ -47,7 +48,7 @@ <h3> Comment Section</h3>
                         <ul class="oldComments" id=1></ul>
                     </div>
                     <div class="postComment">
-                        <form action="/comments" method="POST">
+                        <form action="/comments" method="POST" style ="display:none;">
```

Comment:
Is it just hiding form if user unlogged? If yes - it's very easy to avoid by setting browser inspector values...

Reply:
yes, it hides the post comment text input from user if they aren't signed in.

Reply:
There could be security breach... But for now it's fine